extensions/wikia/BannerNotifications/BannerNotificationsController.class.php

```diff
@@ -5,7 +5,7 @@
  * @author Maciej Brencz
  */
 
-class NotificationsController extends WikiaController {
+class BannerNotificationsController extends WikiaController {
```

Comment:
I'm not sure if this class should be renamed. It's also responsible for displaying Oasis notification: https://one.wikia-inc.com/wiki/Oasis/Notifications_and_confirmations